CONTRIBUTING.md

```diff
@@ -70,20 +77,40 @@ We have a sophisticated CI process setup which gets run on every PR. You must pa
 Once your changes are ready, you can raise a PR. The title of your PR should match the following format:
 
 ```text
-<tag>(<package>): <short description>
+<type>(<package>): <Subject> (#Issue Number)
+```
+
+Samples: (even more [samples](https://github.com/typescript-eslint/typescript-eslint/commits/main))
```

Comment:
Linking to the latest commits is a good idea. But this sentence is a little oddly worded. Maybe...

```suggestion
You can find more samples of good past PR titles in [recent commits to `main`](https://github.com/typescript-eslint/typescript-eslint/commits/main))
```

Related: it's an accessibility necessity to have informative link titles. When screenreader folks navigate through the page by skipping through links, they don't receive context around the links. Titles like _"samples"_ don't make much sense out of context.